util/Migrator/DbScripts/2020-11-18_00_EmergencyAccess.sql

```diff
@@ -0,0 +1,399 @@
+/*
+ * Add support for Emergency Access
+ */
+CREATE TABLE [dbo].[EmergencyAccess] (
```

Comment:
Migration script should be re-runnable; please add a check before creating the table, `IF NOT EXISTS`; can also include the trailing FK key creation, etc.